backend/db/utils.py

```diff
@@ -57,9 +97,21 @@ def chunk_list(input_list: List, chunk_size) -> List[List]:
         yield input_list[i:i + chunk_size]
 
 
-def current_datetime():
-    """Get current datetime"""
-    return datetime.now(timezone.utc).isoformat()
+def current_datetime(time_zone=['UTC/GMT', 'EST/EDT'][1]) -> str:
```

Comment:
FYI: New util that creates a timestamp that is compatible with the Enclave API. It must be a very detailed ISO formatted datetime. I've set the hour offset default to GMT.